backend/langchain_config.py

```
@@ -4,14 +4,21 @@
 """
 
 import os
+import warnings
+# Suppress LangChain JSONB deprecation warning until LangChain updates
+warnings.filterwarnings('ignore', category=DeprecationWarning, module='langchain.*')
+warnings.filterwarnings('ignore', message='Please use JSONB instead of JSON for metadata')
 from typing import Optional, Dict, Any
```

Ruff (0.14.1)

`typing.Dict` is deprecated, use `dict` instead

(UP035)